debian/qubes-core-agent.preinst

```diff
@@ -51,6 +51,10 @@ if [ "$1" = "install" ] ; then
     }
     usermod -L -a --groups qubes user
 
+    ## Allow passwordless login for user "user" (when using 'sudo xl console').
+    ## https://github.com/QubesOS/qubes-issues/issues/1130
+    usermod --password '' user
```

Comment:
This is already done in upgrade case below. Maybe it should be unified, like that call below moved outside of `if`?

Reply:
Yes, better. For simplicity and less bugs (at cost of some more needless output) I think it is always good to avoid using install and upgrade cases.